scripts/xacro.py

```diff
@@ -694,16 +724,19 @@ def main():
         print()
     else:
         eval_self_contained(doc)
-        banner = [xml.dom.minidom.Comment(c) for c in
-                  [" %s " % ('=' * 83),
-                   " |    This document was autogenerated by xacro from %-30s | " % args[0],
-                   " |    EDITING THIS FILE BY HAND IS NOT RECOMMENDED  %-30s | " % "",
-                   " %s " % ('=' * 83)]]
+        banner = [
+            xml.dom.minidom.Comment(c) for c in [
+                " %s " %
+                ('=' * 83), " |    This document was autogenerated by xacro from %-30s | " %
+                args[0], " |    EDITING THIS FILE BY HAND IS NOT RECOMMENDED  %-30s | " %
+                "", " %s " %
+                ('=' * 83)]]
```

Comment:
This is much uglier now.

Reply:
Well this is PEP8 taking the lead here. It's more tight in terms of length though.

Reply:
I am going to at least fix this line as it was more readable before.